codalab/worker/worker_run_state.py

```diff
@@ -645,8 +738,20 @@ def _transition_from_FINALIZING(self, run_state):
         move on to the FINISHED state. Can also remove bundle_path now.
         """
         if run_state.is_restaged:
+            log_bundle_transition(
+                run_state.bundle.uuid,
+                run_state.stage,
+                RunStage.RESTAGED,
+                'the bundle is going to be sent back to the server',
```

Comment:
For what case does restaged occur? I think that should be explained.

Reply:
comes from here - https://github.com/codalab/codalab-worksheets/blob/master/codalab/worker/worker_run_state.py#L644

Reply:
I think the reason why the bundle is being sent back should be included in the log message.

Reply:
I don't know why exactly the bundle is being sent back as it's commented by Jing.

Reply:
I know why exactly is it sent back to the server as it's commented by Jing. It comes from this PR - https://github.com/codalab/codalab-worksheets/pull/2451 where I saw your approval.

Reply:
Can you include that in the reason? The `reason` parameter should explain why the bundle is being restaged.